crates/mev-share-rpc-api/src/types.rs

```diff
@@ -16,10 +16,16 @@ pub struct SendBundleRequest {
     /// The transactions to include in the bundle.
     #[serde(rename = "body")]
     pub bundle_body: Vec<BundleItem>,
+    /// Requirements for the bundle to be included in the block. 
+    #[serde(rename = "validity", skip_serializing_if = "Option::is_none")] 
+    pub validity_predicate: Option<ValidityPredicate>,
+    /// Preferences on what data should be shared about the bundle and its transactions
+    #[serde(rename = "privacy", skip_serializing_if = "Option::is_none")]
+    pub privacy_predicate: Option<PrivacyPredicate>,
```

Comment:
should we consider dropping the predicate suffix and mirror the spec 1:1 `privacy: Option<Privacy>`?

imo this would be cleaner

Reply:
agree, removed 👍